Dev-Corner/tutorials/data_source_integration.md

```diff
@@ -3,26 +3,19 @@ Data Source Integration
 
 :grey_question: ***Questions***
 
-- *Major question that would be addressed in this tutorial (mostly general biological questions)*
-- *Second question*
-- *Third question*
-- *...*
+- *Is it possibile to import data into Galaxy from an external database?*
+- *Is available any ready-to-use example?*
```

Comment:
* How can I write a tool that can import data into Galaxy from an external database?
* What are `data sources` and how do they function?
* Is there any ready-to-use example?